src/tests/test_jsproxy.py

```diff
@@ -389,65 +392,36 @@ def test_mount_object(selenium):
         let b = { x : x2, y, u : 3, t : 7};
         pyodide.registerJsModule("a", a);
         pyodide.registerJsModule("b", b);
-        return pyodide.runPython(`
+        let result_proxy = pyodide.runPython(`
             from a import x
             from b import x as x2
             result = [x(), x2()]
             import a
             import b
             result += [a.s, dir(a), dir(b)]
             result
-        `).toJs()
+        `);
+        let result = result_proxy.toJs()
+        result_proxy.destroy();
+        return result;
         """
     )
     assert result[:3] == ["x1", "x2", 3]
     assert set([x for x in result[3] if len(x) == 1]) == set(["x", "y", "s", "t"])
     assert set([x for x in result[4] if len(x) == 1]) == set(["x", "y", "u", "t"])
-
-
-@pytest.mark.xfail
-def test_mount_map(selenium):
```

Comment:
It's redundant with existing tests?

Reply:
It tests for behavior that was never implemented and that I'm not really planning to implement. Note the `xfail` marker.